src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -196,10 +194,47 @@ public void deleteAttachment(String requestId, String companyNumber, String obje
                 () -> new AttachmentNotFoundException(String.format(ATTACHMENT_NOT_FOUND_MESSAGE, attachmentId))
         );
 
+        Map<String, Object> logMap = buildLogMap(companyNumber, objectionId);
+        deleteFromS3(requestId, attachmentId, logMap);
+
         attachments.remove(attachment);
 
         objection.setAttachments(attachments);
 
         objectionRepository.save(objection);
+
+    }
+
+    private void deleteFromS3(String requestId, String attachmentId, Map<String, Object>  logMap) throws ServiceException {
```

Comment:
I've added another comment related to this, but I don't see the point in the logMap parameter. I don't think it's needed.

Reply:
Think this is now resolved - logMap will only contain the attachmentId.